npm-scripts.mjs

```diff
@@ -445,7 +445,9 @@ function installMsysMake()
 		pythonPath = String(res.stdout).trim();
 	}
 
-	executeCmd(`${pythonPath} worker\\scripts\\getmake.py`);
+	const dir = path.resolve('worker/out/msys');
+
+	executeCmd(`${pythonPath} worker\\scripts\\getmake.py --dir="${dir}"`);
```

Comment:
cosmetic, this path could aslo be created with `path.resolve()` and avoid the double back slashes.

Reply:
Yes, but we don't follow an specific approach for that in the whole file. I agree we need consistency but I don't want to decide such a consistency within this PR.